package.json

```diff
@@ -23,71 +23,72 @@
+    "@types/jest": "^29.4.0",
+    "@types/luxon": "^3.2.0",
+    "@types/node": "^18.13.0",
+    "@types/react": "^18.0.27",
+    "@types/react-dom": "^18.0.10",
     "@types/testing-library__jest-dom": "^5.14.5",
-    "@typescript-eslint/eslint-plugin": "^5.41.0",
-    "@typescript-eslint/parser": "^5.41.0",
-    "axios": "^0.26.1",
+    "@typescript-eslint/eslint-plugin": "^5.51.0",
+    "@typescript-eslint/parser": "^5.51.0",
+    "axios": "^1.3.2",
     "babel-eslint": "^10.1.0",
-    "babel-jest": "^27.5.1",
+    "babel-jest": "^29.4.2",
     "babel-plugin-transform-async-to-promises": "^0.8.18",
-    "babel-preset-solid": "^1.5.4",
-    "bundlewatch": "^0.3.2",
-    "concurrently": "^7.1.0",
+    "babel-preset-solid": "^1.6.10",
+    "bundlewatch": "^0.3.3",
+    "concurrently": "^7.6.0",
     "current-git-branch": "^1.1.0",
-    "eslint": "7.x",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-config-react-app": "^5.2.1",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-config-standard-react": "^9.2.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-flowtype": "5.x",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "6.x",
+    "eslint": "8.34.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-config-react-app": "^7.0.1",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-config-standard-react": "^13.0.0",
+    "eslint-import-resolver-typescript": "^3.5.3",
+    "eslint-plugin-flowtype": "8.0.3",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "7.20.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-standard": "^5.0.0",
     "git-log-parser": "^1.2.0",
-    "jest": "^27.5.1",
+    "jest": "^29.4.2",
+    "jest-environment-jsdom": "^29.4.2",
     "jest-fail-on-console": "^3.0.2",
     "jsonfile": "^6.1.0",
-    "luxon": "^2.3.2",
-    "prettier": "^2.6.2",
+    "luxon": "^3.2.1",
+    "prettier": "^2.8.4",
     "prettier-plugin-svelte": "^2.9.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.70.2",
-    "rollup-plugin-size": "^0.2.2",
+    "rimraf": "^4.1.2",
+    "rollup": "^3.15.0",
+    "rollup-plugin-size": "^0.3.1",
     "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-visualizer": "^5.6.0",
+    "rollup-plugin-visualizer": "^5.9.0",
     "solid-js": "^1.5.7",
     "solid-testing-library": "^0.3.0",
     "stream-to-array": "^2.3.0",
-    "ts-jest": "^27.1.1",
-    "ts-node": "^10.7.0",
-    "typescript": "4.7.4",
-    "vue": "^3.2.33"
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1",
+    "typescript": "4.9.5",
```

Comment:
One remark: I think we agreed on coding with the TS version that is the lowest that we'll also support, which is 4.7.4. This will avoid accidentally using a newer TS feature like `satisfies` that would then break for users.